util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -234,8 +247,11 @@ macro (seqan_build_system_init)
     # settings for development mode
     if ((SEQAN_BUILD_SYSTEM STREQUAL "DEVELOP") AND
         (NOT MSVC) AND
-        (NOT SEQAN_NO_NATIVE))
+        (NOT SEQAN_NO_NATIVE) AND
+        (NOT SEQAN_32BIT_TARGET_PLATFORM))
```

Comment:
It seems too stringent. Some people may want to make CPU-optimized binaries in 32bit machine.
However, I think it's just enough for this time.